### PR TITLE
Create EuanSFotheringham

### PR DESCRIPTION
**Pull request by esfoth**
Hello?

### CODE REVIEW

**Comment by esfoth** [comment]
wooh

**Comment by esfoth** [comment]
woah man, woah

File: EuanSFotheringham [hunk 1]
```diff
@@ -0,0 +1 @@
+EuanSFotheringham
```
Comment (esfoth) [inline EuanSFotheringham]: edit?

**Comment by WilliamZerai** [comment]
bra woa

**Pull request merged**